composables/users.ts

```diff
@@ -70,7 +68,7 @@ const publicInstance = ref<ElkInstance | null>(null)
 export const currentInstance = computed<null | ElkInstance>(() => currentUser.value ? instanceStorage.value[currentUser.value.server] ?? null : publicInstance.value)
 
 export function getInstanceDomain(instance: ElkInstance) {
-  return instance.accountDomain || withoutProtocol(instance.uri)
+  return instance.accountDomain ?? instance.domain
```

Comment:
We use the stored instance from local storage before fetching. We probably need to support where we can reading from the old instance format from local storage. I just got an error when checking the PR the first time I open it, then it worked as the new config was cached. Maybe this is ok, but we should review there is no chance of messing with the user config. I think the safest is to still support `instance.uri` as a fallback here, and check other places too.